mindmup-editabletable.js

```diff
@@ -67,7 +70,6 @@ $.fn.editableTableWidget = function (options) {
 			editor.hide();
 		}).keydown(function (e) {
 			if (e.which === ENTER) {
-				setActiveText();
```

Comment:
why was this removed? it seems useful to be able to edit a cell by hitting 'enter'

Reply:
This is a bug in the code. When the enter key is pressed this also causes the blur event to fire. The table has an event hook on blur to call `setActiveText` so we want to avoid calling it twice.